lib/ex_gdax.ex

```diff
@@ -222,10 +222,11 @@ defmodule ExGdax do
 
   ## Examples
 
-      iex> ExGdax.cancel_orders()
+      iex> ExGdax.cancel_orders(params)
       {:ok, ["XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"]}
   """
   defdelegate cancel_orders, to: ExGdax.Private, as: :cancel_orders
+  defdelegate cancel_orders(order_id), to: ExGdax.Private, as: :cancel_orders
```

Comment:
How do you feel about calling this `cancel_order` instead of `cancel_orders`?

Reply:
Agreed, I merged #4 with `cancel_order`